client/src/components/ExplorerTransactions.vue

```diff
@@ -1,299 +1,357 @@
+			<br>
+		</div>
+		<div v-if="!isSpinnerActive && transactions" class="box">
+			<div class="container" v-for="(transaction,key,index) in transactions" v-bind:key="key">
+				<transaction v-if="progressive_display_level>index" :tx_id="key" :transaction="transaction"
+										 :no_border="index == (Object.keys(transactions).length-1)" :about_wallet_ids="redirected_ids"/>
+			</div>
+			<!--			<div class="py-3 main-block">-->
+			<!--				<div class="text-center">-->
+			<!--					<b-pagination-->
+			<!--							v-model="currentPage"-->
+			<!--							:total-rows="count_total"-->
+			<!--							per-page="20"-->
+			<!--							@change="onPageChanged"-->
+			<!--							size="l"-->
+			<!--							class="pl-4 pt-2 my-0"-->
+			<!--					></b-pagination>-->
+			<!--					-->
+			<!--				</div>-->
+			<!--			</div>-->
+		</div>
+	</div>
 </template>
 
 <script>
-import Transaction from './ExplorerTransaction.vue'
-import validate from 'bitcoin-address-validation';
-import EditWalletModal from './commons/EditWalletModal.vue';
-import Exchange from './commons/Exchange.vue';
-import BtcAmount from './commons/BtcAmount.vue';
-import WalletId from './commons/WalletId.vue';
-const conf = require("../conf.js");
+	import Transaction from './ExplorerTransaction.vue'
+	import validate from 'bitcoin-address-validation'
+	import EditWalletModal from './commons/EditWalletModal.vue'
+	import Exchange from './commons/Exchange.vue'
+	import BtcAmount from './commons/BtcAmount.vue'
+	import WalletId from './commons/WalletId.vue'
+	import { ModalProgrammatic } from 'buefy'
 
-export default {
+	const conf = require('../conf.js')
+
+	export default {
 		components: {
 			Transaction,
-			EditWalletModal,
+			// EditWalletModal,
 			Exchange,
 			BtcAmount,
-			WalletId
+			WalletId,
 		},
 		props: {
 			request_input: {
 				type: String,
-				required: true
+				required: true,
 			},
 			page: {
 				type: Number || String,
 				required: false,
-				default: 1
-			}
-	},
-	data() {
-		return {
-			transactions: null,
-			count_total: null,
-			exchangeName: null,
-			exchange :null,
-			wallet_id: null,
-			exchangeWallets: null,
-			title: null,
-			isSpinnerActive: true,
-			failoverText: null,
-			redirected_ids: [],
-			isRemoving: null,
-			walletOwner: null,
-			walletIdToEdit: null,
-			tx_id: null,
-			total_on_wallets: null,
-			addr_count: null,
-			progressive_display_level: 1,
-			timerId: null
-		}
-	},
-	watch: {
-		$route(route) {
-			this.currentPage = this.page || 1;
-			this.getTransactions();
-		}
-	},
-	created() {
-		this.currentPage = this.page || 1;
-		this.getTransactions();
-		this.timerId = setInterval(() => {
-			this.progressive_display_level++;
-		}, 200);
-	},
-	beforeDestroy(){
-		clearInterval(this.timerId);
-	},
-	methods: {
-		updateTitleAndDescription(){
-			if (this.wallet_id){
-				document.title =  this.$t("explorerTransactionsPageTitleWalletId", {wallet_id: this.wallet_id, website_name: conf.website_name});;
```

Comment:
@fenrirheimr, please suggest your lead to integrate VueMeta. It helps to update title, description and meta tags on the fly using computed properties.